srcalc/src/main/java/gov/va/med/srcalc/web/view/admin/EditRule.java

```diff
@@ -58,11 +74,16 @@ public EditRule(final AdminService adminService, final Rule rule)
             fMatchers.add(ValueMatcherBuilder.fromPrototype(matcher));
         }
         fSummandExpression = rule.getSummandExpression();
-        fBypassEnabled = rule.isRequired();
+        fBypassEnabled = rule.isBypassEnabled();
         fDisplayName = rule.getDisplayName();
         fNewVariableKey = null;
     }
     
+    public void setAdminService(final AdminService adminService)
```

Comment:
Same thought about calling `buildNew()` without calling this method. Can `buildNew()` just take the AdminService as a parameter?